exporters/scss/src/js/generators/simple.ts

```diff
@@ -91,8 +91,8 @@ export function generateSimple(
     } else if (token.tokenType === 'Radius') {
       value = printUnit(token.value.radius.measure, token.value.radius.unit);
     } else if (token.tokenType === 'GenericToken') {
-      const unitLessProp = token.properties.find((prop) => prop.codeName === 'unitless');
-      if (unitLessProp && !!unitLessProp.value) {
+      const unitLessProp = token.propertyValues.unitless;
+      if (unitLessProp && !!unitLessProp) {
```

Comment:
Wasn't it enough just use double negation? cc @literat

Reply:
Tru